lib/driver.js

```diff
@@ -1781,57 +1776,15 @@ class XCUITestDriver extends BaseDriver {
     }
   }
 
-  /**
-   * Get session capabilities merged with what WDA reports
-   * This is a library command but needs to call 'super' so can't be on
-   * a helper object
-   */
-  async getSession() {
```

Comment:
Sounds good, we should still keep the endpoint itself as inspector usage, but no need to return WDA specific, afaik